.github/workflows/polyglot_release.yml

```diff
@@ -18,29 +18,26 @@ jobs:
       run: |
         pip install twine
         twine upload --skip-existing -u __token__ -p ${{ secrets.PYPI_TOKEN }} dist/*
-  
-  build_mac_os:
+  macos_build:
```

Comment:
Can u ponder over the idea of adding another workflow to release from [older ubuntu versions](I would https://github.com/actions/runner-images)?
We want to specifically support debian 10 => ubuntu 18/20 idk. 

https://pypi.org/project/polyglot-piranha/0.3.0/#files These are our current releases for ubuntu. We have two of them. `_34` was produced from this `ubuntu-latest` architecture. But to support all internal uses cases we need to support `_28` architecture too (which i had manually released to Pypi from an internal debian 10 machine) . So, let's see if running on ubuntu 20/18 gives us this `_28` wheel.

Then the release will become fully automated . We can make a release daily. lol

Reply:
resolved